tests/cupyx_tests/scipy_tests/special_tests/test_polygamma.py

```diff
@@ -42,6 +44,9 @@ def test_scalar(self, xp, scp, dtype):
         return scp.special.polygamma(
             dtype(2.), dtype(1.5)).astype(numpy.float32)
 
+    @pytest.mark.xfail(
+        platform.processor() == "aarch64",
+        reason="aarch64 scipy does not match cupy/x86 see Scipy")
```

Comment:
Is this meant to have an issue #?

Reply:
Sorry, I think it was https://github.com/scipy/scipy/issues/20159
Will update it